infracost-usage-example.yml

```diff
@@ -473,6 +473,10 @@ resource_usage:
   azurerm_mariadb_server.my_server:
     additional_backup_storage_gb: 2000 # Additional consumption of backup storage in GB.
 
+  azurerm_mssql_database.my_database:
+    vcore_hours: 600 # Number of used vCore-hours for serverless compute.
+    monthly_long_term_retention_storage_gb: 1000 # Number of GBs used by long-term retention backup storage.
```

Comment:
This was my bad, we should rename these `monthly_vcore_hours` since it's not continuous (resets at the start of each month) and `long_term_retention_storage_gb` since it is continuous.